packages/httpsnippet-client-api/src/index.ts

```diff
@@ -87,6 +87,7 @@ const client: Client<APIOptions> = {
     title: 'API',
     link: 'https://npm.im/api',
     description: 'Automatic SDK generation from an OpenAPI definition.',
+    extname: '.js',
```

Comment:
This is a new requirement I added in `@readme/httpsnippet` when I upgraded it to support ESM.